.circleci/config.yml

```diff
@@ -119,10 +116,10 @@ workflows:
   version: 2
   all:
     jobs:
-    - doc:
+    - doc
```

Comment:
You missed a trailing colon here: `doc:`

Reply:
oops